app/controllers/admin/organizations_controller.rb

```diff
@@ -9,7 +9,7 @@ def update
     if OrganizationUpdateService.update(@organization, organization_params)
       redirect_to admin_organizations_path, notice: "Updated organization!"
     else
-      flash[:error] = "Failed to update this organization."
+      flash[:error] = @organization.errors.collect { |error| "#{error.attribute}: " + error.message }.join("<br />".html_safe)
```

Comment:
Is there a way to render the errors on the top when rendering the `:edit`, we would likely be able to include some logic in the template to handle showing the errors in cleaner format. Flash messages start becoming hard to read if it were to be a single line (which I believe the <br /> tries to address.

Reply:
I had grabbed this as an example from another controller, but I agree. I think I'll revert this and update the template to display the error at the appropriate form input.